tests/0143-exponential_backoff.c

```diff
@@ -149,10 +148,9 @@ int helper_exponential_backoff(rd_kafka_mock_cluster_t *mcluster,
                                         high = (retry_max_ms * 125) / 100;
                                 if (low > ((retry_max_ms * 75) / 100))
                                         low = (retry_max_ms * 75) / 100;
-                                if ((time_difference < high) &&
-                                    (time_difference > low)) {
-                                        error_code = -3;
-                                        break;
+                                if ((time_difference > high) &&
```

Comment:
This condition is still wrong. 

time_difference > high || time_difference < low

It's mathematically impossible for time_difference > high && time_difference < low if high > low.

Reply:
sorry my bad. I will fix it and run it then push